lib/ssf/ssf_methods.rb

```diff
@@ -23,6 +23,7 @@ def finish(time: nil)
     end
 
     def child_span(operation: '', tags: {})
+      puts "WARN: this method is deprecated."
```

Comment:
That's true, but I still feel like it might not be a bad idea to leave it for a bit jut in case.